go/pkg/cas/upload_test.go

```diff
@@ -145,6 +145,12 @@ func TestFS(t *testing.T) {
 			},
 			wantScheduledChecks: []*uploadItem{rootWithoutAItem, bItem, subdirItem, cItem},
 		},
+		{
+			desc:                "root-without-a-using-allowlist",
+			paths:               []*PathSpec{{Path: filepath.Join(tmpDir, "root"), Allowlist: []string{"b", "subdir"}}},
```

Comment:
Add a check where only `subdir/c` is included? 

May also be nice to have a slightly more complicated one like `dir/f2`, `dir/sub/f2` are added whereas in there are more files in those subfolders.

Reply:
Done